src/plone/restapi/configure.zcml

```diff
@@ -38,6 +38,15 @@
       for="Products.CMFPlone.interfaces.ITestCasePloneSiteRoot"
       />
 
+  <genericsetup:registerProfile
+      name="testing-workflows"
+      title="plone.restapi testing-workflows"
+      directory="profiles/testing-workflows"
+      description="Adds sample workflows for testing"
+      provides="Products.GenericSetup.interfaces.EXTENSION"
+      for="Products.CMFPlone.interfaces.ITestCasePloneSiteRoot"
+      />
+
```

Comment:
the profile must be excluded in https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/setuphandlers.py#L21

Reply:
@deiferni this need to be resolved before merge.

Reply:
@jensens i have amended the desired changes in https://github.com/plone/plone.restapi/blob/aa99141eeaeb742aa7c1852d48d373c09d1a26b1/src/plone/restapi/setuphandlers.py#L21 yesterday, but i am currently fighting with what to me seem to be unrelated test-failures. will request re-review when it is ready!